lib/ipmi_backend_utils.pm

```diff
@@ -164,22 +164,25 @@ sub umount_installation_disk {
     assert_script_run("ls $mount_point");
 }
 
-#Get the partition where the new installed system is installed to
+# Get the partition where the new installed system is installed to
 sub get_installation_partition {
     my $partition = '';
 
-    #Do not use script_output because when the command fail, script_output dies
-    type_string(qq{fdisk -l | grep "^/dev/sda.*\\*" | cut -d ' ' -f 1 | tee /dev/$serialdev\n});
-    $partition = wait_serial;
-    $partition =~ s/^\s+|\s+$//g;
-    save_screenshot;
-    if (is_storage_ng && ($partition eq '')) {
-        record_soft_failure "bsc#1080729 - Partitioner does not mark boot flag";
-        my $y2log_file                = '/var/log/YaST2/y2log';
-        my $root_partition_commit_msg = script_output(qq{grep 'Commit Action "Adding mount point / of .* to /etc/fstab' $y2log_file});
-        $root_partition_commit_msg =~ m{Commit Action "Adding mount point / of ([\S]*) to /etc/fstab}m;
-        $partition = $1;
+    # Confirmed with dev that the reliable way to get partition for / is via installation log, rather than fdisk
+    # For details, please refer to bug 1101806.
+    my $cmd        = '';
+    my $y2log_file = '/var/log/YaST2/y2log';
+    if (sle_version_at_least('15')) {
```

Comment:
can you please replace this with `is_sle('15+')` ? as far as I know we trying to deprecate `sle_version_at_least` and unify all checks with `is_sle` instead. @okurz  can you confirm?

Reply:
Correct. There is an open PR already to change all existing code so I would appreciate if you use is_sle already everywhere

Reply:
Got it. Will replace with is_sle.

I will check if is_storage_ng works for me.

Reply:
Actually is_storage_ng and is_sle('15+') currently seems have the same effect. And the code logic is to use version to determine behavior. So if you agree, I would prefer using is_sle('15+').

Reply:
Sure, there is not so much difference. Personally I prefer to use checks like "is_storage_ng" when we understand that not just for single versions something differs but because of new features that were introduced. Then we would also cover cases which have an effect on other distributions, e.g. openSUSE, caasp/kubic, etc. In this case I think the code path is only used for the SLE virtualization tests on bare-metal so it's fine like it is